meerk40t/kernel/kernel.py

```diff
@@ -1865,16 +1865,20 @@ def process_queue(self, *args) -> None:
                 if signal in self.listeners:
                     listeners = self.listeners[signal]
                     removed = False
+                    ct = 0
                     for i, listen in enumerate(listeners):
+                        ct += 1
                         listen_funct, listen_lso = listen
                         if (listen_funct == remove_funct or remove_funct is None) and (
                             listen_lso is remove_lso or remove_lso is None
                         ):
                             del listeners[i]
                             removed = True
                             break
-                    if not removed:
-                        print("Value error removing: %s  %s" % (str(listeners), signal))
+                    if not removed and ct>0:
```

Comment:
This is just notifying of the error it's not the error itself. --- But, I haven't really located the problem. This is just saying that it tried to remove that twice and it did but that's an issue with the lifecycle not the notice here.

Reply:
I'll hunt down the real error later.